_data/projects/trapteam-cc.yml

```
@@ -0,0 +1,12 @@
+name: Trap Team CC
+desc: A website and backend service for pest trappers
+site: trapteamnz.tk/git.html
```

The last build failure is because this URL doesn't match the expected format. This should fix it:

```suggestion
site: https://www.trapteamnz.tk/git.html
```